var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -47,6 +47,12 @@ class PyJaxlib(PythonPackage, CudaPackage):
     depends_on("py-absl-py", when="@:0.3", type=("build", "run"))
     depends_on("py-flatbuffers@1.12:2", when="@0.1", type=("build", "run"))
 
+    conflicts(
+        "+cuda cuda_arch=none",
```

Comment:
```suggestion
        "cuda_arch=none",
        when="+cuda",
```
This is actually probably clearer and matches what we do for PyTorch